youths/schema/mutations.py

```diff
@@ -246,43 +246,45 @@ class UpdateYouthProfileInput(YouthProfileFields):
 class UpdateYouthProfileMutation(relay.ClientIDMutation):
     class Input:
         id = graphene.Argument(graphene.ID, required=True)
-        profile = UpdateYouthProfileInput(required=True)
+        youth_profile = UpdateYouthProfileInput(required=True)
+
+    youth_profile = graphene.Field(YouthProfileNode)
 
     @classmethod
     @staff_required
     @transaction.atomic
     def mutate_and_get_payload(cls, root, info, **input):
-        input_data = input.get("profile")
+        input_data = input.get("youth_profile")
 
         youth_profile = YouthProfile.objects.get(pk=from_global_id(input.get("id"))[1])
         youth_profile = update_youth_profile(
             input_data, youth_profile, manage_permission=True
         )
-        return UpdateMyYouthProfileMutation(profile=youth_profile)
+        return UpdateMyYouthProfileMutation(youth_profile=youth_profile)
```

Comment:
Is there a reason why this is not returning an instance of `UpdateYouthProfileMutation`?

Reply:
That is clear error, I'll fix it.